_posts/2023-10-26-louisvillejefferson-county-metro-government-balance-kentucky/louisvillejefferson-county-metro-government-balance-kentucky.Rmd

```diff
@@ -0,0 +1,135 @@
+
+This finding suggests that the reduced competitiveness in Louisville ultimately appears to contribute to creating favorable conditions for the incumbent, resulting in a 2-seat advantage.
+
+## Methods and Materials
+
+### Methods
+
+I sampled 175,000 districting plans for Louisville across two independent runs of the Sequential Monte Carlo (SMC) algorithm. I performed the analysis above with 5,000 samples from one of the SMC runs.
+
+#### Pre-processing Notes
+
+1.   Excluded 32 precincts[^5] for which geographical data were unavailable.
+
+[^5]: The excluded precincts are: A141, A143, A155, B186, E186, E187, G197, H146, H161, H170, H172, H174, I143, J142, J143, J161, J167, K136, M187, M190, M204, M205, M206, M207, M208, M209, N124, N127, N159, O139, Q147, V135.
+
+<!-- -->
+
+2.   Included 5 elections, Mayor, US Senator, County Judge/Executive, County Clerk, and Coroner to calculate the estimated baseline partisanship.
+
+3.   Combined all geographical data of neighborhoods, municipalities and places into one dataset, removing any overlaps.
+
+![Figure 4: Neighborhoods used for redistricting simulations. Since Louisville has merged with Jefferson County in 2003, there are three types of perceived local communities: Neighborhoods(yellow), Municipalities(green), Places(blue).](images/local_ties.png){fig-align="center" width="80%"}
+
+### Materials
+
+#### Data
+
+-   Election Data
+
+    -   General elections in Louisville 2022 (available at [Jefferson County Clerk Election Center](https://elections.jeffersoncountyclerk.org/election-results/))
+
+-   Geographical Data
+
+    -   [Precincts](https://data.louisvilleky.gov/datasets/LOJIC::jefferson-county-ky-voting-precincts/explore?location=38.189642%2C-85.678250%2C11.32)
+    -   [Districts](https://hub.arcgis.com/datasets/3fcc3da3573e4511a45c19a4e880b05b/explore)
+    -   [Neighborhoods, Municipalities, and Places](https://www.lojic.org/neighborhoods)
+
+#### VRA Compliance
+
+![Figure 5: The graph shows minority voting age population share, with districts ordered from smaller share to larger share. Each point represents a simulated district, with blue indicating that the district is majority Democratic, and red indicating a majority Republican.](images/minority_vap.png){fig-align="center" width="80%"}
+
+The figure above shows that Louisville already has approximately five Black opportunity-to-elect districts. Therefore, I did not impose additional constraints for simulations in this regard. I have also conducted a validation for the VRA Compliance in the simulated plans, and it has been confirmed that VRA Compliance is guaranteed in these plans.
```

Comment:
Could you add additional detail on what you mean by this? 

> I have also conducted a validation for the VRA Compliance in the simulated plans, and it has been confirmed that VRA Compliance is guaranteed in these plans.

I would also avoid the phrase "VRA Compliance is guaranteed," since that is a legal question we won't really know for sure here.